src/python/pants/backend/python/target_types_test.py

```diff
@@ -338,35 +338,74 @@ def test_parse_requirements_file() -> None:
     }
 
 
-def test_python_distribution_dependency_injection() -> None:
+def test_inject_python_distribution_dependencies() -> None:
```

Comment:
renamed test to follow naming pattern of `test_inject_pex_binary_entry_point_dependency`, which I found more descriptive. (I prefer test names that closely relates to the function under test ;) )